lib/commands/context.js

```diff
@@ -212,9 +212,10 @@ async function setupNewChromedriver (opts, curDeviceId, adbPort) {
     adbPort
   };
   let chromedriver = new Chromedriver(chromeArgs);
+  let appPackage = (opts.chromeOptions && opts.chromeOptions.androidPackage) || opts.appPackage;
```

Comment:
@jlipps 
Thank you for the comment.
As per https://sites.google.com/a/chromium.org/chromedriver/getting-started/getting-started---android
Android-specific Desired Capabilities setting androidPackage solved the issue, but as you said if chromeAndroidPackage can also be set by client then yes, I think same needs to be handled.